app/graphql/types/query_type.rb

```diff
@@ -1,23 +1,25 @@
 module Types
   class QueryType < Types::BaseObject
+    # Return all markets
     field :all_markets, [Types::MarketType], null: false
     def all_markets
       Market.all
     end
-
-    # Save for future use to search markets by products, etc.
-    # 
-    # field :all_markets, [Types::MarketType], null: false
-    # def all_markets
-    #   Market.all
-    # end
-
+    # Return a single market by id
     field :market, Types::MarketType, null: false do
       argument :id, Integer, required: true
     end
-    def market(id)
-      Market.find(id[:id])
+    def market(id:)
```

Comment:
This is the syntax I was talking about. Actually read about it in this Ruby book I'm reading. It's cool because order doesn't matter, it just tells the method to look through the incoming data for a key of `id`

Reply:
Nice that makes it much easier to digest